controller_manager/package.xml

```diff
@@ -10,17 +10,18 @@
   <buildtool_depend>ament_cmake</buildtool_depend>
 
   <build_depend>ament_index_cpp</build_depend>
-  <build_depend>class_loader</build_depend>
   <build_depend>controller_interface</build_depend>
   <build_depend>hardware_interface</build_depend>
+  <build_depend>pluginlib</build_depend>
```

Comment:
Let's not bundle everything into this PR, here's a follow-up ticket: https://github.com/ros-controls/ros2_control/issues/46

Reply:
Fixed when rebasing against master anyways.